tests/FlexBridgeRabbitMqTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare(strict_types=1);
+
+namespace Platformsh\FlexBridge\Tests;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+use Platformsh\ConfigReader\Config;
+use function Platformsh\FlexBridge\mapPlatformShRabbitMq;
+
+class FlexBridgeRabbitMqTest extends TestCase
+{
+    /**
+     * @var MockObject|Config
+     */
+    private $configMock;
+
+    public function setUp(): void
+    {
+        $this->configMock = $this->getMockBuilder(Config::class)
```

Comment:
Please use the pattern set in the other tests.  We are not using the PHPUnit mock library.